tests/jenesis/config/test_parsing.py

```diff
@@ -59,19 +63,192 @@ def test_init_create_project():
     input_file_name = "jenesis.toml"
     path = os.path.join(os.getcwd(), input_file_name)
     with open(path, encoding="utf-8") as toml_file:
-        toml_dict = toml.load(toml_file)
+        data = toml.load(toml_file)
 
-    assert toml_dict["project"]["name"] == os.path.basename(os.getcwd())
+    assert data["project"]["name"] == os.path.basename(os.getcwd())
 
     user_name = subprocess.getoutput("git config user.name")
     user_email = subprocess.getoutput("git config user.email")
     authors = [f"{user_name} <{user_email}>"]
 
-    network = {"name": network_name}
+    network = {"name": ""}
     network.update(vars(fetchai_testnet_config()))
 
-    assert toml_dict["project"]["authors"] == authors
-    assert toml_dict["profile"]["testing"]["network"] == network
+    assert data["project"]["authors"] == authors
+    assert data["profile"]["testing"]["network"] == network
+
+    os.remove("jenesis.toml")
+    shutil.rmtree("contracts")
+
+
+def test_add_profile():
```

Comment:
It seems a bit unusual to add all these other test cases to `config/test_parsing.py`. Maybe better to separate the tests for each command into their own files? I'd suggest:
```
tests/jenesis/cmd/test_add.py
tests/jenesis/cmd/test_compile.py
tests/jenesis/cmd/test_deploy.py
...
```

Reply:
The new structure is better, but I still think it would be best to put the command-related tests (basically everything but `test_parsing`) under `jenesis/tests/cmd/` rather than `jenesis/tests/config/`.